### ISSUE TITLE
Hotel related icons

### ISSUE THREAD

**Issue by pcout** [lucide-icons/lucide#683]
Can you please add hotel related icons ?
Thanks.

**Comment by karsa-mistmere** [comment]
@pcout: Can you please provide a list and use cases?

**Comment by pcout** [comment]
Well, I was hoping all the related features an hotel has.
I've attached some examples found in a quick google search.
Thanks.

![image](https://user-images.githubusercontent.com/687515/170289729-4b9f79ac-caca-4da4-9afc-5b7f917242df.png)
![image](https://user-images.githubusercontent.com/687515/170289877-995e1b3e-5084-495b-b6c9-4d27ffc6deb8.png)
![image](https://user-images.githubusercontent.com/687515/170289952-50197a26-8d38-485a-85d5-cb340e1eb18e.png)

**Comment by ericfennis** [comment]
@pcout Could be nice to have list to work with. Also to check which icons are relevant for the icon set. We can't just add them all. Some are too specific.

**Comment by pcout** [comment]
Here a list. We can think of more.
-------------------------------------------------------------------------
Hotel
Guest (man)
Guest (woman)
2 guests
2 ou more guests
Baby
Child
Reception bell
Receptionist
Clock
Mini fridge
Bathtube
Bath cabin
Lamp(s)
Room key
Key card
Single bed (front view)
Single bed (side view)
King size bed (front view)
King size bed (side view)
Door sign
AVAC
Air conditioner
Suitcase
Luggage
Calendar (wih booking)
Calendar (wihout booking)
Cocktail
Swimming pool
Some sports (horse riding, golf, tennis, swimming etc)
-------------------------------------------------------------------------

**Comment by karsa-mistmere** [comment]
These ones we already have, I believe:
- [x] Hotel: not sure if adding a separate hotel icon is feasible, probably just use [`building`](https://lucide.dev/icon/building) or [`building-2`](https://lucide.dev/icon/building)
- [x] Guest (man/woman): just use [`user`](https://lucide.dev/icon/user), there's an [ongoing discussion](https://github.com/feathericons/feather/pull/46) about adding gender icons but it's a complex issue.
- [x] 2 guests, 2 or more guests [`users`](https://lucide.dev/icon/users) or [`user-plus`](https://lucide.dev/icon/user-plus)
- [x] Clock: [`clock`](https://lucide.dev/icon/clock) or [`alarm-clock`](https://lucide.dev/icon/alarm-clock)
- [x] Bathtub: [`bath`](https://lucide.dev/icon/bath)
- [x] Room key: [`key`](https://lucide.dev/icon/key)
- [x] Key card: [`credit-card`](https://lucide.dev/icon/credit-card)
- [x] Calendar (without booking): [`calendar`](https://lucide.dev/icon/calendar)
- [x] Cocktail: use `martini` once #674 is approved and merged.
- [x] Swimming pool: I think using [`waves`](https://lucide.dev/icon/waves) is feasible

These are ones I think would be good to have:
- [ ] Baby: but not quite sure if we really need this.
- [ ] Reception bell
- [ ] Lamp(s)
- [ ] Single bed (either front or side view, but not both)
- [ ] King size bed (either front or side view, but not both)
- [ ] Suitcase: already have a few drafts for this & luggage.
- [ ] Luggage
- [ ] Air conditioner: but I guess [`thermometer-snowflake`](https://lucide.dev/icon/thermometer-snowflake) or [`snowflake`](https://lucide.dev/icon/snowflake) may also be used.

These are ones I think could be generalized a bit:
- [ ] Mini fridge: far too generic, a refrigerator icon could be nice though
- [ ] Bath cabin: seems a bit too specific, we could maybe add a shower head icon to represent it?
- [ ] AVAC: maybe a fan icon could work and is broad enough?
- [ ] Calendar (with booking): seems too specific, but some extra calendar icons could be nice to have (`calendar-plus`, `calendar-check`, `calendar-off`, `calendar-x` come to mind)

These ones seem either too specific or too generic to me:
- [ ] Child: we don't really have adult human being icons to begin with.
- [ ] Receptionist: could be represented with concierge bell if and when it's added.
- [ ] Door sign: seems too specific for me. What would the use case be?
- [ ] Some sports (horse riding, golf, tennis, swimming etc): these seem a tad out of scope but I'll see what I can come up with.

**Comment by karsa-mistmere** [comment]
Here are a few ideas for some of the icons:
![image](https://user-images.githubusercontent.com/17746067/170523050-ed0c9de7-7161-44dc-b4d9-1330572e7d2f.png)
![hotel](https://user-images.githubusercontent.com/17746067/170523157-558b73bc-00e5-4a54-9144-3c7d019fcf9e.png)

I'm not sure we need this many lamps though. :D

**Comment by pcout** [comment]
Ok. I thinks it´s looking good. :) 
Keep the lamps. :D

**Comment by karsa-mistmere** [comment]
@pcout: Could you provide a use case for each of the lamp icons?

**Comment by pcout** [comment]
> @pcout: Could you provide a use case for each of the lamp icons?

Corridors lamps, Bedrooms lamps, common area lamps, lobby lamps...

**Comment by pcout** [comment]
As for the baby/child icons:
the guests can be adults plus baby and/or more children. 
I can definitely see several use cases for this.

**Comment by karsa-mistmere** [comment]
> > @pcout: Could you provide a use case for each of the lamp icons?
> 
> Corridors lamps, Bedrooms lamps, common area lamps, lobby lamps...

I know what they represent, but what would the use case be? On what kind of component would these icons be present?

**Comment by pcout** [comment]
> > > @pcout: Could you provide a use case for each of the lamp icons?
> > 
> > 
> > Corridors lamps, Bedrooms lamps, common area lamps, lobby lamps...
> 
> I know what they represent, but what would the use case be? On what kind of component would these icons be present?

infographics, maps, figma layouts, etc...

**Comment by karsa-mistmere** [comment]
For example? Sorry I'm being dense but I really can't quite see what these could be needed for.

**Comment by pcout** [comment]
icons for Home Assistan, for sure. 
![image](https://user-images.githubusercontent.com/687515/170744036-fdfb119f-5483-4fcc-a2dc-c86e66e05f6b.png)

**Comment by karsa-mistmere** [comment]
![image](https://user-images.githubusercontent.com/17746067/170930998-71a9df4b-014e-4f86-97eb-b6b696db7185.png)
